composer.json

```diff
@@ -1,31 +1,50 @@
 {
   "name": "wordpress/performance",
-  "type": "wordpress-plugin",
-  "license": "GPL-2.0-or-later",
   "description": "Performance plugin from the WordPress Performance Team, which is a collection of standalone performance features.",
-  "homepage": "https://wordpress.org/plugins/performance-lab/",
+  "license": "GPL-2.0-or-later",
+  "type": "wordpress-plugin",
   "keywords": [
     "performance",
     "wordpress"
   ],
+  "homepage": "https://wordpress.org/plugins/performance-lab/",
   "support": {
     "issues": "https://github.com/WordPress/performance/issues"
   },
+  "require": {
+    "php": "^7.2 || ^8.0",
+    "ext-json": "*"
+  },
   "require-dev": {
-    "wp-phpunit/wp-phpunit": "^5.8",
+    "phpcompatibility/php-compatibility": "^9.3",
+    "phpstan/extension-installer": "^1.3",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-deprecation-rules": "^1.1",
+    "phpstan/phpstan-phpunit": "^1.3",
+    "slevomat/coding-standard": "^8.0",
+    "squizlabs/php_codesniffer": "^3.9",
+    "szepeviktor/phpstan-wordpress": "^1.3",
+    "wp-coding-standards/wpcs": "^3.1",
+    "wp-phpunit/wp-phpunit": "^6.5",
     "yoast/phpunit-polyfills": "^1.0"
   },
-  "require": {
-    "composer/installers": "~1.0",
-    "ext-json": "*",
-    "php": ">=7|^8"
+  "autoload-dev": {
+    "psr-4": {
+      "PerformanceLab\\Tests\\": "tests/utils"
+    }
+  },
+  "config": {
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "phpstan/extension-installer": true
```

Comment:
Is this missing `composer/installers`? I'm getting the following when I do `composer install`:

![image](https://github.com/WordPress/performance/assets/134745/2ff0e4cd-32c4-4379-bcd9-07a16a362191)

Reply:
```
composer/installers contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "composer/installers" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] ?
y - add package to allow-plugins in composer.json and let it run immediately
n - add package (as disallowed) to allow-plugins in composer.json to suppress further prompts
d - discard this, do not change composer.json and do not allow the plugin to run
? - print help
Do you trust "composer/installers" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] d
```

I went with `d`. I'm not seeing it happen now when I do `composer install`. 😕

Reply:
This might be due to stale vendor content. I don't see if we are using this plugin so I removed it.